ledger/participant-integration-api/src/main/scala/platform/apiserver/RateLimitingInterceptor.scala

```diff
@@ -33,7 +28,8 @@ private[apiserver] final class RateLimitingInterceptor(
     memoryMxBean: GcThrottledMemoryBean,
 ) extends ServerInterceptor {
 
-  private val logger = LoggerFactory.getLogger(getClass)
+  private implicit val logger: ContextualizedErrorLogger =
+    DamlContextualizedErrorLogger.forClass(getClass)
```

Comment:
Probably not addressable in this PR, but here we don't populate the correlation id/ submission id since it's readily available. It would be beneficial to look into options for populating this value so the participant operator/LAPI client can more easily trace heavy requests (that trip the participant on big load). What do you think? CC @mziolekda

Reply:
Well, for that to happen we would have to decode the message i.e. let it through, that's exactly what we don't want to do